package.json

```diff
@@ -391,6 +391,11 @@
 					"type": "boolean",
 					"default": false,
 					"description": "Whether to automatically send a 'hot reload' request during a Flutter debug session when saving files."
+				},
+				"dart.analyzeAngularTemplates": {
+					"type": "boolean",
+					"default": false,
```

Comment:
I just re-read this and realised this plug isn't in the SDK, you need to built it yourself. I'm wondering if for now this should be a hidden option so it doesn't show up in the settings completion - otherwise I worry people will turn it on expecting it to work and it won't! I've started listing "Preview Features" in the [release notes](https://github.com/Dart-Code/Dart-Code/releases/tag/v2.3.3) so maybe it makes sense there for now?